accname/name/comp_labelledby.html

```diff
@@ -31,7 +31,7 @@ <h2 id="h">div group label</h2>
 -->
 
 <script>
-AriaUtils.verifyLabelsBySelector(".ex");
+AriaUtils.verifyLabelsBySelector(1, ".ex");
```

Comment:
See below... Keep `comp_labelledby.html` as-is (but with a warning) to make way for https://github.com/web-platform-tests/wpt/pull/42522

Reply:
#42522 was since merged so it's no longer necessary to leave this change out of the PR, but it should get the mentioned argument order change, and will need an update to the `expectedCount` integer.